README.md

```diff
@@ -80,26 +85,34 @@ cp -i config/example_config.yml ~/.dp-cli-config.yml
 vi ~/.dp-cli-config.yml
 ```
 
-[ set paths for:
+update the paths and ssh-user: 
 
 ```yaml
-    dp-setup-path:
-    dp-hierarchy-builder-path:
-    dp-code-list-scripts-path:
-
-   set your `ssh-user:`
+    dp-setup-path: path to your local dp-setup
+    dp-ci-path: path to your local dp-ci
+    dp-hierarchy-builder-path: path to your local dp-hierarchy-builder-path
+    dp-code-list-scripts-path: path to your local dp-code-list-scripts-path
+    ssh-user: Your AWS account name
```

Comment:
I don't think AWS account name is clear, as people sign in with an email address rather than a username. We should be explicit here and just say something like "Your first and last name concatenated eg. JaneBloggs"